cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -232,7 +232,17 @@ func (c *CVCController) updateCVCObj(
 // 4. Create cstorvolumeclaim resource.
 // 5. Update the cstorvolumeclaim with claimRef info and bound with cstorvolume.
 func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis.CStorVolumeClaim, error) {
-	_ = cvc.Annotations[string(apis.ConfigClassKey)]
+
+	policyName := cvc.Annotations[string(apis.VolumePolicyKey)]
+	volumePolicy := &apis.CStorVolumePolicy{}
+	var err error
+	if policyName != "" {
+		klog.Infof("uses cstorvolume policy for volume configuration")
```

Comment:
Better to log CVC name and CVP name as well which will be useful for debugging purose.

Reply:
done